tests/integration/adapter/test_singular.py

```diff
@@ -0,0 +1,10 @@
+from dbt.tests.adapter.basic.test_singular_tests import BaseSingularTests
+from dbt.tests.adapter.basic.test_singular_tests_ephemeral import BaseSingularTestsEphemeral
+
+
+class TestSingularTests(BaseSingularTests):
```

Comment:
Why does it contain https://github.com/ClickHouse/dbt-clickhouse/pull/129/?

Reply:
Because originally I wasn't going to release the #129 PR since it just had tests, but the bug fix seemed important to release.  I closed 129.